src/components/CoordinatorHome.js

```diff
@@ -4,149 +4,209 @@ import styled from "styled-components"
+        }
+      ]
+    },
+    {
+      Header: 'Reported Data',
+      columns: [
+        {
+          // TODO: Link up to the correct value
+          Header: "Took Medication",
+          accessor: "took_medication"
+        },
+        {
+          // TODO: put side effects into a line break list
+          Header: "Side Effects",
+          accessor: "side_effects",
+          Cell: e=> e.value.join("\n \n")
+        },
+        {
+          Header: "Photo",
+          accessor: "photo",
+          Cell: e=>
+            <PhotoPopout>
+              <Image
+                    src={strip_report}
+                    alt={"Strip report"}
+                    onClick={(e) => {
+                      e.preventDefault()
+                      e.stopPropagation()
+                    }}
+              />
+            </PhotoPopout>
+        },
+        {
+          Header: 'Test Result',
+          // TODO: link each checkbox to each patient. Right now we are routing all data to 
+          // setPhotoStatus and not capturing for whom
+          accessor: 'status',
+          width: 175,
+          Cell: observer(e=> <div>
+                  <label>
+                    {/* TODO: try changing to a radio button and making the result of the change equal
+                    to the opposite */}
+                    {/* onChange={e => store.setPhotoStatus('positive')} */}
+                    {/* store.current_strip_report = 'negative' */}
+                    <Input type="checkbox" name="photostatus" onChange={e => store.current_strip_report = 'positive'}
+                           checked={store.current_strip_report === 'positive'} /> Positive
+                    <br></br>
+                    <Input type="checkbox" name="photostatus" onChange={e => store.current_strip_report = 'negative'} 
+                          checked={store.current_strip_report === 'negative'} /> Negative / Unclear
+                    <br></br>
+                    {/* TODO: style like a button */}
+                    <Input type="submit" name="photostatus" onClick={e => (store.setPhotoStatus(store.current_strip_report))}/>
+                  </label>
+                </div>)
+        },
+        {
+          Header: "Contact",
+          accessor: "phone",
+          // Add's a What's App Link to the table
+          // TODO: make sure phone number is properly formatted in DB
+          Cell: e =><a href={'https://wa.me/' + e.value} target="_blank"> {e.value} </a>
+        },
+        {
+          Header: "Profile",
+          accessor: "id",
+          Cell: e=><a hrer={e.value} target="_black"> View </a>
+        },
+        {
+          Header: "My Notes",
+          accessor: "coordinator_note",
+          width: 175,
+          Cell: e=>
+          <div>
+            <Input use="textarea" />
+            <br></br>
+            <br></br>
+            <Button>Save Note</Button>
+          </div>
+          // TODO: Add a submit button
```

Comment:
Is this submit button still planned?

`src/components/CoordinatorNote.js`

Reply:
Yes, I added a TODO comment focused on connecting the button with an onclick that makes an entry into the DB